ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -314,85 +133,6 @@ describe('AppsPanel', () => {
     expect(findAvailableApps(wrapper, false).exists()).toBeFalsy();
   });
 
-  it('should pause a running RStudio app', async () => {
```

Comment:
These are not like the "getAppsByDisplayGroup" tests above - are there equivalents for these tests elsewhere? @evrii